libmscore/system.cpp

```diff
@@ -39,6 +39,8 @@
 #include "chordrest.h"
 #include "iname.h"
 #include "spanner.h"
+#include "sym.h"
+#include "symbol.h"
```

Comment:
Could be.  Seems I wouldn't have added it if I hadn't gotten a compilation error, but maybe I was confused.